css/sticky.scss

```diff
@@ -0,0 +1,109 @@
+
+.sticky__heading{
+    text-align: center;
+    color: var(--secondary-700);
+    font-size: 2.4rem;
+    font-weight: 700;
+}
+
+#sticky-notes-container{
+    display: flex;
+    padding: 2rem ;
+    border: 2px solid gray;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    min-height: 100vh;
+    gap: 2rem;
+ 
+}
+
+.sticky__card{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 2rem;
+    /* border: 2px solid palevioletred; */
+    border-radius: 0.5rem;
+    min-width: 320px;
+    max-width: 320px;
+    min-height: 300px;
+    max-height: 300px;
```

Comment:
Why not use rem here too? Please check the other instances as well.

Reply:
Will change it ✨